install/usable.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * @package Polylang
+ *
+ * /!\ THE CODE IN THIS FILE MUST BE COMPATIBLE WITH PHP 5.6.
+ *
+ * /!\ THE CONSTANTS `POLYLANG`, `PLL_MIN_PHP_VERSION`, AND `PLL_MIN_WP_VERSION` MUST BE DEFINED.
+ */
+
+/**
+ * Class tat can tell if Polylang can be activated.
+ *
+ * @since 3.8
+ */
+class PLL_Usable {
```

Comment:
Similar to above for re-usability, what do you think to introduce 3 methods:
```
get_min_php_version()
get_min_wp_version ()
get_plugin_name()
```

Reply:
Done in [8c09281](https://github.com/polylang/polylang/pull/1704/commits/8c09281f707e4bbb8860c7ca65b45ed3e517dd92).